### ISSUE TITLE
No retransmission request on packet loss

### ISSUE THREAD

**Issue by a-andre** [eulynx-live/librasta#83]
To reproduce the issue either configure only one channel in the rasta cfg files or configure virtual network interfaces and disable one channel by setting the packet loss to 100% for both directions. And set the packet loss for one direction of the remaining channel to for example 10%. (See script below).

On packet loss, instead of a retransmission request, there is a disconnection after Tmax.

The problem seems to be that `red_f_deferTmo()` is never called. It should be
called, when "There is a message waiting in the deferQueue, which has
been already stored there for longer than Tseq."

## Example script to setup network configuration
```
ip netns add ns1

ip link add vth1 netns ns1 type veth peer vth1
ip link add vth2 netns ns1 type veth peer vth2

ip netns exec ns1 ip addr add 1.1.1.1/24 dev vth1
ip addr add 1.1.1.2/24 dev vth1

ip netns exec ns1 ip addr add 1.1.2.1/24 dev vth2
ip addr add 1.1.2.2/24 dev vth2

ip netns exec ns1 ip link set dev vth1 up
ip link set dev vth1 up

ip netns exec ns1 ip link set dev vth2 up
ip link set dev vth2 up

# disable channel 2
ip netns exec ns1 tc qdisc add dev vth2 root netem loss 100%
tc qdisc add dev vth2 root netem loss 100%

# add some loss to channel 1 of server
ip netns exec ns1 tc qdisc add dev vth1 root netem loss 10%
```

**Comment by laugengebaeck** [comment]
Yeah, that is weird. If you like, feel free to contribute a fix. Otherwise, I cannot guarantee when someone of us will have time to take a look at this, because librasta is unfortunately not our priority.

**Comment by laugengebaeck** [comment]
As far as I can see, 359db19 seems to be the culprit. Maybe it helps to restore `channel_timeout_event()` to the version before that?
